lib/email/styles.rb

```diff
@@ -121,8 +121,10 @@ def onebox_styles
       style('aside.quote .avatar', 'margin-right: 5px; width:20px; height:20px; vertical-align:middle;')
       style('aside.quote', 'border-left: 5px solid #e9e9e9; background-color: #f8f8f8; margin: 0;')
 
-      style('blockquote', 'border-left: 5px solid #e9e9e9; background-color: #f8f8f8; margin: 0;')
-      style('blockquote > p', 'padding: 1em;')
+      style('blockquote', 'border-left: 5px solid #e9e9e9; background-color: #f8f8f8; margin-left: 0; padding: 12px;')
+      style('blockquote > :first-child', 'margin-top: 0 !important;')
```

Comment:
I'm obligated to ask if there's any way you can do this without `!important`?

Reply:
We're inlining these styles anyway, so having it `important!` free doesn't matter that much, but yeah - moving stuff around makes it so that these styles are applied first, and then all the other ones are then prepended so these take precedence.